index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Amr Ahmed - Portfolio</title>
+</head>
+<body>
+    <img src="/images/top_circles.png" alt="Background image circles white">
+    <!-- <img src="/images/portfoliobg.png" alt="Background image circles white"> -->
+    <div class="wrapper">
+        <div class="header">
+            <span>Amr Ahmed</span> <div class="hamburger-menu"><i class="fas fa-bars"></i></div>
+        </div>
+        <div class="headline-section">
+            <span class="myName">
+                Amr Ahmed
+            </span>
+            <span class="role">Software Developer</span>
+            <div class="self-intro">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hesitate to contact me.</div>
+        </div>
+    </div>
```

Comment:
- [ ] Please change the `<div>` container and instead use the `<section>` tag to wrap the content in the main body. This improves the semantics of your HTML.